xml/depl_maintenance.xml

```diff
@@ -924,8 +924,8 @@
       </step>
       <step>
        <para>
-        Check for other (custom) repositories. All &slsa; SP2 repositories need
-        to be replaced with the respective &slsa; SP3 version. In case no SP3
+        Check for other (custom) repositories. All &slsa; SP3 repositories need
+        to be replaced with the respective &slsa; SP4 version. In case no SP3
```

Comment:
Should this be "In case no SP4 version exists..." (just going by the previous version)